resources/github-comment-markdown.template

```diff
@@ -49,7 +49,7 @@
 </details>
 
 <!-- TEST RESULTS IF ANY-->
-<% if(testsErrors != null && testsErrors instanceof List && testsErrors?.any{item -> item?.status == "FAILED"}) {%>
+<% if(testsErrors?.any{item -> item?.status == "FAILED"}) {%>
```

Comment:
After merging https://github.com/elastic/apm-pipeline-library/pull/746 is not required (https://github.com/elastic/apm-pipeline-library/pull/493/files#r415694615)